File: Java/program/Human.java
class Human{
    private String NIK;
    private String nama;
    private String jenis_kelamin;
    Human(){
       
      }
      //setter and getter
    public String getNIK(){
        return this.NIK;
    }
       public String getNama(){
        return this.nama;
    } 
        public String getJenkal(){
        return this.jenis_kelamin;
    }
     public void setNIK(String NIK){
         this.NIK=NIK;
    }
       public void setNama(String nama){
         this.nama=nama;
    } 
        public void setJenkal(String jenis_kelamin){
         this.jenis_kelamin=jenis_kelamin;
    }

}

File: PHP/Program/Human.php
<?php

class Human{
// private atribut

private $NIK = "";
private $nama = "";
private $jenis_kelamin= "";

function __construct($NIK="",$nama="",$jenis_kelamin="",) { // CONSTRUCTOR

$this->NIK = $NIK;
$this->nama = $nama;
$this->jenis_kelamin = $jenis_kelamin;

}
 # Getter and Setter

function setNIK ($NIK="") {
    $this->NIK = $NIK;
}
function setnama ($nama="")  {
    $this->nama= $nama;
     
}
    function setjenis_kelamin($jenis_kelamin="") {
        $this->jenis_kelamin= $jenis_kelamin;
    }

  

    function getNIK() {
    
    return $this->NIK;
    }
    function getnama () {
        return $this->nama;
        }
    function getjenis_kelamin() {
        
        return $this->jenis_kelamin;
        }

}
?>